iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/DatabasePartitionTable.java

```diff
@@ -593,6 +594,12 @@ public List<Integer> getDataRegionIds() {
     return dataRegionIds;
   }
 
+  public Stream<TConsensusGroupType> getRegionType(int regionId) {
```

Comment:
I believe it will be better if return **List** instead of Stream (to keep consistent with other method).
And, getRegionType**s**